doc/reference/tt_cli/index.rst

```diff
@@ -0,0 +1,29 @@
+tt CLI utility
+==========
+
+``tt`` is a utility that provides a unified command-line interface for managing
+Tarantool instances and Tarantool-based applications. It covers a wide range of
```

Comment:
`Tarantool instances and Tarantool-based applications` sounds strange. I think that `for managing Tarantool-based applications` is enough.